tests/test_marshall.py

```diff
@@ -58,118 +54,52 @@ def assert_no_accidental_overwrites(self, instance, data, key):
 
 class TestCSVMarshall(MarshallInterfaceTests):
     @pytest.fixture
-    def instance(self, tmpdir):
-        return CSVMarshall(pathlib.Path(tmpdir))
+    def regression_root(self, tmpdir):
+        return RegressionRoot(tmpdir, data_format='csv')
+
+    @pytest.fixture
+    def instance(self):
+        return CSVMarshall
```

Comment:
You're returning a class instead of an instance here. Or maybe CSVMarshall is a singleton now?

Reply:
CSVMarshall (and the other marshalls) are effectively classes used just as namespaces.  I don't have any mechanisms for hard enforcement of singletons, but the intention is to use the classes directly (all methods are class methods).